draft-ietf-quic-multipath.md

```diff
@@ -351,31 +314,40 @@ does not discuss when a client decides to initiate a new path. We
 delegate such discussion in separate documents.
 
 To let the peer open a new path, an endpoint needs to provide its peer with connection IDs
-and associated Path Identifiers for the new path.
+for at least one unused path identifier. Endpoints SHOULD use the MP_NEW_CONNECTION_ID
+frame to provide new connection IDs and, respectively, the MP_RETIRE_CONNECTION_ID frame to
+retire connection IDs after a successful handshake indicating multipath support by both endpoints.
```

Comment:
I think we should clarify that we discuss Path ID 0 here. By design, proposing new connection IDs on other Path IDs is only possible using MP_NEW_CONNECTION_ID.

Reply:
see below. Okay?